python/cugraph/cugraph/tests/traversal/test_paths.py

```diff
@@ -37,13 +38,29 @@
 DISCONNECTED_GRAPH = CONNECTED_GRAPH + "8,9,4"
 
 
+paths_test_data = {
+    "shortest_path_length_1_1": 0,
+    "shortest_path_length_1_5": 2.0,
+    "shortest_path_length_1_3": 2.0,
+    "shortest_path_length_1_6": 2.0,
+    "shortest_path_length_-1_1": ValueError,
+    "shortest_path_length_1_10": ValueError,
+    "shortest_path_length_0_42": ValueError,
+    "shortest_path_length_1_8": 3.4028235e38,
+}
+
+
+@pytest.fixture(scope="module")
+def load_traversal_results():
+    return load_resultset("traversal", None)
```

Comment:
same comment as above, I don't think this needs to return anything, same for all others like this.